### DISCUSSION TITLE
About the web page redirection problem in GitHub student certification

### DISCUSSION THREAD

**Discussion by lipu0324** [community/community#86583] [GitHub Education]
### Hi everyone,

There are some strange problems here. When I opened the github education page, he told me that I already have the student package.
<img width="1125" alt="image" src="https://github.com/community/community/assets/72327270/fb5ceeff-f98c-41a0-9bc6-59c573c8c026">
But in fact this is not the case. For example, when I use the Copilot plug-in in Vscode, it prompts me that I do not have permission to access the copilot service.
<img width="912" alt="image" src="https://github.com/community/community/assets/72327270/f5a55261-fbff-4d9f-a756-6993b3e6b184">
What happens next is this, when I click "submit your information again" or "reverify your academic affiliation", I will be redirected to "https://github.com/pricing"
<img width="1374" alt="image" src="https://github.com/community/community/assets/72327270/3f5d0e97-a50f-4eff-ab87-497f7f2ef943">
I don't know if this is because there is some problem with my account or if there is something wrong with the education verification step. Maybe someone has encountered the same problem or has a solution?

**Comment by sronald01** [comment]
It's the same issue as described in https://github.com/orgs/community/discussions/86486

It is definitely a problem at GitHub Education team's side and I'm quite disappointed this issue gets suppressed by support as described in  the linked issue and I find it weird no more people report issues like this one.

**Comment by ZheruiHuang** [comment]
Same issue here. It's disappointing that I reported this issue last November and have seen more and more people reporting this issue over the months without getting any response from the github team.

**Comment by CarmJos** [comment]
Is there any updates?

**Comment by huskyhao** [comment]
same issue! Not fixed yet!

**Comment by fahmikdp** [comment]
sameeeee

**Comment by Kobe972** [comment]
Why is the support team silent? Even if they say it’s a bug, that’s fine

**Comment by maddox05** [comment]
bro still happening

**Comment by zhanghulu2004** [comment]
The same issue
